### DISCUSSION TITLE
Monitor inputs (LED) and report status

### DISCUSSION THREAD

**Discussion by geraldFS** [arendst/Tasmota#13881] [Q&A] [answered]
Hi there,

I have a question, is there a possibility that Tasmota picks up and forwards signals?

I have a heating control that has two push button inputs that switch them
1x hot water preparation (manual)
1x heating operation
Their operation is signaled by two LEDs.

In other words, if the boiler is empty outside the time, or if there is a purely increased need for hot water (BATHTUB), you press the hot water button. LED lights up. The hot water boiler is heated to the target temperature.
Signaling with LED, if you press it again, the hot water generation goes off - the LED is then off.

Two LEDs must be tapped and forwarded to the home assistant.
I place two buttons on incoming outputs.

But how does the evaluation and dissemination of the LED do?

I want to use a WEMO D1 as hardware, possibly optocoupler, etc.

Gerald

**Comment by sfromis** [comment, marked as answer]
If you want to make electrical connections, you should be very sure of the voltages and electrical behavior of the heating controller. It is possible that a device not designed for connections is doing "surprising" things here. The safest technique, without analyzing the heating controller, would be to use light sensors to sense LED on vs off. This is similar to adding optocouplers to the controller, except that then you need to be sure of voltages.

An analog light sensor is not great here, as ESP8266 only has one analog input. ESP32 does have more. One possible I2C light sensor, already supported (up to two sensors) by precompiled tasmota-sensors, is BH1750.
